_topic_maps/_topic_map_ms.yml

```diff
@@ -142,6 +142,8 @@ Topics:
   File: expanding-persistent-volumes-microshift
 - Name: Dynamic storage using the LVMS plugin
   File: microshift-storage-plugin-overview
+- Name: Creating volume snapshots
```

Comment:
This section talks about creating, restoring, and deleting. Maybe the section title should be more general? "Managing ..." or "Working with ..." for example?